src/module.ts

```diff
@@ -71,6 +71,15 @@ export default defineNuxtModule<ModuleOptions>({
       }
     })
     runtimeConfig.hub = hub
+    // Make sure to tell Nitro to not generate the wrangler.toml file
+    // @ts-expect-error nitro.cloudflare.wrangler is not yet typed
+    delete nuxt.options.nitro.cloudflare?.wrangler?.compatibility_flags
```

Comment:
I would suggest:

- Make a copy of whole `wrangler` object
- Set it to an empty object
- In a way that is safe for hub, respect it's options.

Reply:
updated!